packages/@ourworldindata/grapher/src/mapCharts/MapChart.tsx

```diff
@@ -216,9 +216,29 @@ export class MapChart
     }
 
     @action.bound onDocumentKeyDown(e: KeyboardEvent): void {
-        // hide the globe on hitting the Escape key
+        // Hide the globe on hitting the Escape key
         if (e.key === "Escape" && this.mapConfig.globe.isActive) {
             this.globeController.hideGlobe()
+            this.globeController.resetGlobe()
+            this.mapConfig.region = MapRegionName.World
+        }
+    }
+
+    @computed get externalLegend(): HorizontalColorLegendManager | undefined {
```

Comment:
One thing I'm noticing with the external legend: When faceted, the hovered-over bin isn't being highlighted, whereas it is for the unfaceted variant.

<img width="938" height="299" alt="Image" src="https://github.com/user-attachments/assets/3fdf7153-8d97-4fb0-bb96-70ec9533ea02" />

<img width="938" height="299" alt="Image" src="https://github.com/user-attachments/assets/eb61e2c1-5eb4-4884-b5b7-179e62fab8d8" />

Reply:
Good catch, fixed!